cogs/music.py

```
@@ -0,0 +1,581 @@
+            pass
+    
+    async def check_empty_voice_channels(self):
+        """Check for empty voice channels and leave them."""
+        await self.bot.wait_until_ready()
+        
+        while not self.bot.is_closed():
+            for guild in self.bot.guilds:
+                voice_client = guild.voice_client
+                if voice_client and voice_client.is_connected():
+                    # Check if there are any non-bot members in the voice channel
+                    if not any(member for member in voice_client.channel.members if not member.bot):
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)